tests/getRaw.py

```diff
@@ -25,8 +25,9 @@
 import os
 
 import unittest
-import lsst.utils.tests as utilsTests
+import lsst.utils
```

Comment:
Please separate functional changes (like whether testdata_decam is setup) and style changes (like what the module is called when it's imported) into separate commits.